src/basic_bot/commons/mjpeg_video.py

```diff
@@ -1,11 +1,16 @@
 import asyncio
+from typing import TYPE_CHECKING
 
 from aiohttp import web, MultipartWriter, client_exceptions
 import cv2
 
 from basic_bot.commons import log
 from basic_bot.commons.base_camera import BaseCamera
 
+if TYPE_CHECKING:
```

Comment:
I wonder why this was neccessary here when everywhere(?) else it just added type imports without regard for TYPE_CHECKING?